src/components/mixins/InputDebounce.js

```diff
@@ -20,29 +20,22 @@ export default {
        * Checking if the event it is a string, if it is a truthy value, and it doesn't have
        * the property target in it
        * */
-      if (window.ProcessMaker && window.ProcessMaker.debounce) {
-        if (typeof event === 'string' && !!event && event.target === undefined) {
-          this.touched = true;
-          return this.updateValue(event);
-        }
+      let useDebounce = window.ProcessMaker && (window.ProcessMaker.debounce === undefined || window.ProcessMaker.debounce === true)
```

Comment:
from our convo, I though we were going to go away from the debounce property inside `window.ProcessMaker`

Reply:
also, we are mixing types. In some places debounces is being declared as arrays, then here is being declared as boolean type, which one is it